src/styles/components/modal/styles.less

```diff
@@ -1,19 +1,13 @@
 & when (@modal-enabled) {
 
   .modal {
-    z-index: @z-index-modal-container;
```

Comment:
To be honest I do not remember this change, but it works perfectly, even for the `ServerErrorModal`